src/System.Private.CoreLib/shared/Internal/Runtime/CompilerServices/Unsafe.cs

```diff
@@ -71,6 +72,7 @@ public static int SizeOf<T>()
         [Intrinsic]
         [NonVersionable]
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        [return: NotNullIfNotNull("value")]
```

Comment:
can this use nameof(value)? (it's outside of the method so not sure if that's allowed)

Reply:
Unfortunately, no.